dashboard/config/locales/scripts.en.yml

```diff
@@ -390,3 +390,28 @@ en:
           description_short: Create a story or make a game with Infinity Play Lab!!
           description: Create a story or make a game with Infinity Play Lab!
           Infinity: Infinity
+        20-hour:
```

Comment:
Great question!  @tanyaparker says crowdin will see that the strings have already been translated and translate them automatically (as long as the text is identical).